src/shared/theme.tsx

```diff
@@ -0,0 +1,4 @@
+export const globalTheme: string | 'dark' | 'light' = 'dark';
```

Comment:
```suggestion
export const globalTheme: 'dark' | 'light' = 'dark';
```
Unless you want to be able to set this to just any string you can omit the `string` type.

Reply:
removed string type, this code will be replaced with context code by @Walsker